chef-windows.gemspec

```diff
@@ -0,0 +1,22 @@
+gemspec = eval(IO.read(File.expand_path("../chef.gemspec", __FILE__)))
+
+gemspec.platform = Gem::Platform.new ["universal", "mingw32", nil]
```

Comment:
Hmm - weirdly enough, it does seem to work.  I just assigns nil to the last item.  Ruby is whacky...